docs/stats/lombscargle.rst

```diff
@@ -679,6 +679,8 @@ the periodogram frequencies using a fast Fourier transform. As with the
 ``slow`` method, it can handle data errors and floating mean.
 The scaling is approximately :math:`O[N\log M]` for :math:`N`
 data points and :math:`M` frequencies.
+Due to the approximation, negative powers are produced for frequencies close to zero.
+Hence, it is suggested to use other methods in such cases.
```

Comment:
I think this should be reworded. Negative powers are not produced every time, and they don't only happen at frequencies near zero.

Maybe something like "The fast algorithm trades accuracy for speed, and produces a close approximation to the true periodogram. In particular, you may observe powers less than zero in some cases."